src/components/home/student/AddStudent.js

```diff
@@ -0,0 +1,401 @@
+        </FormControl>
+      </div>
+      <br />
+      <div className="row">
+        {/* MOTHER'S NAME */}
+        <FormControl fullWidth>
+          <TextField
+            fullWidth
+            id="outlined-basic"
+            name="mother_name"
+            label="Mother's Name"
+            variant="outlined"
+            value={values.mother_name}
+            onBlur={handleBlur}
+            onChange={handleChange}
+          />
+          <p className="errorText">{errors.mother_name}</p>
+        </FormControl>
+        {/* FATHER'S NAME */}
+        <FormControl fullWidth>
+          <TextField
+            fullWidth
+            id="outlined-basic"
+            label="Father's Name"
+            name="father_name"
+            value={values.father_name}
+            onBlur={handleBlur}
+            onChange={handleChange}
+            variant="outlined"
+          />
+          <p className="errorText">{errors.father_name}</p>
+        </FormControl>
+      </div>
+      <br />
+      {/* STUDENT'S PHONE NUMBER */}
+      <div className="row">
+        <FormControl fullWidth>
+          <TextField
+            fullWidth
+            id="outlined-basic"
+            label="Student's Phone Number"
+            name="student_phone_number"
+            variant="outlined"
+            value={values.student_phone_number}
+            onBlur={handleBlur}
+            onChange={handleChange}
+          />
+          <p className="errorText">{errors.student_phone_number}</p>
+        </FormControl>
+        {/* student_email_id ID */}
+        <FormControl fullWidth>
+          <TextField
+            fullWidth
+            id="outlined-basic"
+            label="Student's Email Id"
+            name="student_email_id"
+            variant="outlined"
+            value={values.student_email_id}
+            onBlur={handleBlur}
+            onChange={handleChange}
+          />
+          <p className="errorText">{errors.student_email_id}</p>
+        </FormControl>
+        {/* PARENT'S PHONE NUMBER */}
+        <FormControl fullWidth>
+          <TextField
+            fullWidth
+            id="outlined-basic"
+            label="Parents Phone Number"
+            name="parent_phone_number"
+            variant="outlined"
+            value={values.parent_phone_number}
+            onBlur={handleBlur}
+            onChange={handleChange}
+          />
+          <p className="errorText">{errors.parent_phone_number}</p>
+        </FormControl>
+      </div>
+      <br />
+      <Button
+        sx={{ margin: "1rem", width: "10rem" }}
+        variant="contained"
+        color="success"
+        name="submitButton"
+        type="submit"
+        onClick={handleSubmit}
+      >
+        SUBMIT
+      </Button>
+      <div style={{ margin: "1rem", background: "red" }}>{ }</div>
+      <div style={{ margin: "1rem", background: "green" }}>{ }</div>
+      <Divider orientation="horizontal" flexItem>
+        OR, UPLOAD FILES
```

Comment:
"or" should be bigger in size
```suggestion
        <h3>OR </h3> <br /> UPLOAD FILE
```